### ISSUE TITLE
Form does not open the ticket

### ISSUE THREAD

**Issue by vbenedocci** [pluginsGLPI/formcreator#2480]
**Describe the bug**
Hello. I upgraded from version 2.12.2 to version 2.12.3 and, after the update made by the marketplace, the form that was working stopped working.

When the form is filled, the plugin tries to create a ticket with the ID 0 (zero). In a test database, which had no call, it created it with this id but then doesn't create it anymore, as it tries to create it again as zero.

![image](https://user-images.githubusercontent.com/20477998/143917932-a890e1c0-019f-48d4-bb1f-1ca52a08c217.png)

Another thing, that after the update, the fields defined as text are not stored even in the form responses.

![image](https://user-images.githubusercontent.com/20477998/143918190-42dff753-383e-408f-9910-abbc42eb2c88.png)

**To Reproduce**

**Expected behavior**

**Screenshots**

**GLPI / Plugins (please complete the following information):**
- GLPI version
- list of all plugins (available in GLPI > Setup > General > tab System > bottom of the page)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by btry** [comment]
Hi

Pick fix #2477. Remains to find why a ticket with id 0 is created.

**Comment by vbenedocci** [comment]
Hello.

I think it was some process in the update that went wrong.

I updated two installations, one that is in production and the other that is in testing. For both, the same problem occurred.

To clear up the doubts if it was a bug in the version, I created a completely new installation and the first thing I did was install the plugin, create a form with only one field and link in the ticket and it was opened in the correct ID.
